task/clair-scan/0.1/README.md

```diff
@@ -0,0 +1,25 @@
+# clair-scan task
+
+## Description:
+Perform vulnerability scanning of container images and reports any discovered vulnerabilities.
```

Comment:
Maybe we could also expand on this by mentioning that the Clair scanner is used and linking to the Clair documentation page (https://quay.github.io/clair/whatis.html).

Reply:
ack -  I updated the description to reference Clair and also linked to the documentation. I was conflicted when adding on more sections to the readme than what could potentially get scraped from the yaml but I think have referencing documentation is necessary.

Reply:
Thanks!